From d061e9cc3531796e859babfcfdd7ee3cf90be4a4 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Tue, 9 Jun 2020 10:31:55 +0200
Subject: [PATCH 1/2] Enable watchdog reset

MPSoC watchdog requires PMU support to actually perform a reset. Set the ENABLE_EM
option to enable watchdog.

see:
https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/18842055/Cadence+WDT+Driver
---
 lib/sw_apps/zynqmp_pmufw/src/xpfw_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/sw_apps/zynqmp_pmufw/src/xpfw_config.h b/lib/sw_apps/zynqmp_pmufw/src/xpfw_config.h
index cfef7c3458..9d48c5adda 100644
--- a/lib/sw_apps/zynqmp_pmufw/src/xpfw_config.h
+++ b/lib/sw_apps/zynqmp_pmufw/src/xpfw_config.h
@@ -118,7 +118,7 @@ extern "C" {
 #endif
 
 #ifndef ENABLE_EM_VAL
-#define	ENABLE_EM_VAL						(0U)
+#define	ENABLE_EM_VAL						(1U)
 #endif
 
 #ifndef ENABLE_SCHEDULER_VAL
-- 
2.17.1

